build/odimra/event.service

```diff
@@ -18,7 +18,7 @@ After=network-online.target
 [Service]
 Type=forking
 Restart=always
-ExecStart=/bin/sh -c 'cd /bin ; export CONFIG_FILE_PATH=/etc/odimra_config/odimra_config.json; sudo -E -u odimra nohup ./svc-events --registry=consul --registry_address=consul:8500 --server_address=odimra:45103 --client_request_timeout=`expr $(cat $CONFIG_FILE_PATH | grep SouthBoundRequestTimeoutInSecs | cut -d : -f2 | cut -d , -f1 | tr -d " ")`s >> /var/log/odimra_logs/event.log 2>&1 &'
+ExecStart=/bin/sh -c 'cd /bin ; export CONFIG_FILE_PATH=/etc/odimra_config/odimra_config.json; sudo -E -u odimra nohup ./svc-events --registry=consul --registry_address=consul:8500 --framework=GRPC --server_address=odimra:45103 --client_request_timeout=`expr $(cat $CONFIG_FILE_PATH | grep SouthBoundRequestTimeoutInSecs | cut -d : -f2 | cut -d , -f1 | tr -d " ")`s >> /var/log/odimra_logs/event.log 2>&1 &'
```

Comment:
Please update registry and registry_address with etcd values

Reply:
done